js/views/modals/about/Donations.js

```diff
@@ -1,69 +1,107 @@
-import app from '../../../app';
+import { clipboard } from 'electron';
+import qr from 'qr-encode';
+import { isSupportedWalletCur } from '../../../data/walletCurrencies';
 import { getWallet, launchWallet } from '../../../utils/modalManager';
-import { openSimpleMessage } from '../../modals/SimpleMessage';
 import loadTemplate from '../../../utils/loadTemplate';
 import baseVw from '../../baseVw';
-import { clipboard } from 'electron';
-import qr from 'qr-encode';
 
-const obDonationAddress = '3EN8kP3yW9MGvyiRMtGqKavQ9wYg4BspzR';
-const qrCodeDataURI = qr(`bitcoin:${obDonationAddress}`, { type: 6, size: 6, level: 'Q' });
-const donationCountFloor = 500;
+
 let hiderTimer;
 
 export default class extends baseVw {
   constructor(options = {}) {
+    const opts = {
+      ...options,
+      initialState: {
+        showCoin: 'BTC',
+      },
+      ...options.initialState,
+    };
+
     super({
       className: 'aboutDonations',
-      ...options,
+      ...opts,
     });
+    this.options = opts;
+
+    const btcAddress = '3EN8kP3yW9MGvyiRMtGqKavQ9wYg4BspzR';
+    const bchAddress = 'prws8awqjg8l497x3h7qeu6jps4fdmk75vhlng8akj';
+
+    this.dCoins = {
+      BTC: {
+        obDonationAddress: btcAddress,
+        qrCodeDataURI: qr(`bitcoin:${btcAddress}`,
+          { type: 6, size: 6, level: 'Q' }),
+        supported: isSupportedWalletCur('BTC'),
+      },
+      BCH: {
+        obDonationAddress: bchAddress,
+        qrCodeDataURI: qr(`bitcoin:${bchAddress}`,
+          { type: 6, size: 6, level: 'Q' }),
+        supported: isSupportedWalletCur('BCH'),
+      },
+    };
   }
 
   events() {
     return {
       'click .js-copyAddress': 'copyDonationAddress',
       'click .js-openInWallet': 'openInWalletClick',
+      'click .js-btc': 'showBTC',
+      'click .js-bch': 'showBCH',
     };
   }
 
+  showBTC() {
+    this.setState({ showCoin: 'BTC' });
+  }
+
+  showBCH() {
+    this.setState({ showCoin: 'BCH' });
+  }
+
   copyDonationAddress() {
-    clipboard.writeText(obDonationAddress);
+    const addr = this.dCoins[this.getState().showCoin].obDonationAddress;
+    clipboard.writeText(addr);
 
-    this.$copyNotification.addClass('active');
+    this.getCachedEl('.js-copyNotification').addClass('active');
     if (!!hiderTimer) {
       clearTimeout(hiderTimer);
     }
     hiderTimer = setTimeout(
-      () => this.$copyNotification.removeClass('active'), 3000);
-  }
-
-  get $copyNotification() {
-    return this._$copyNotification ||
-      (this._$copyNotification = this.$('.js-copyNotification'));
+      () => this.getCachedEl('.js-copyNotification')
```

Comment:
Could you store a reference to the `this.getCachedEl('.js-copyNotification')` you add the active class to above and then remove class only on that reference? Otherwise, it's a major edge case, but it looks like if the user switches tabs in-between the animation, the class removal would be on the element in the new tab.

Reply:
Good catch.